website/src/components/HeaderLarge.re

```diff
@@ -123,7 +123,7 @@ let make = (~currentLocation) => {
                accessibilityLabel={item.text}>
                {item.componentFunc(
                   ~iconColor=Consts.Colors.light,
-                  ~iconSize=22.->Svg.Size.pt,
+                  ~iconSize=22.->Svg.pt,
```

Comment:
Are you sure about this? I don't see Svg module being updated.

Reply:
Aargh, search and replace error.